### ISSUE TITLE
Voltage display

### ISSUE THREAD

**Issue by TarasTadva** [OpenI6X/opentx#145]
Problem with battery voltage range. I have a remote control from a 1S battery, but I can not set the range below 4 volts. There is also no way to lower the alarm voltage, less than 4 volts. As a result - a continuous squeak of the buzzer. Can this be fixed in future firmware?

**Comment by ajjjjjjjj** [comment]
On original firmware low voltage is 4.2V. This is still the same hardware and long story short: Altought it may work for you it is not guaranteed to work properly below 4.0V.
As a workaround you can disable battery alarm but i do not recommend it.

**Comment by TarasTadva** [comment]
When working from 1C, a step-up module is used. The equipment operates from its 6 volts, as expected, but the voltage is removed to the voltmeter befor the step-up module, otherwise it will not be possible to track the real battery charge. For this, it is necessary to expand the battery range to 3V and be able to set the alarm to 3V

**Comment by ajjjjjjjj** [comment]
Now i get it. But i'm still thinking if this will not suggest safe voltage for some users.

**Comment by TarasTadva** [comment]
By default, you can leave 4.2 as it was. And only those who need it will change.

**Comment by xpertcrush** [comment]
> By default, you can leave 4.2 as it was. And only those who need it will change.
> 

I agree on this, make range from 3.5 to whatever higher you think will work so alarm can be set as low as 3.5. 

And leave default setup to 4.2 v alarm

**Comment by xpertcrush** [comment]
![P_20220215_204723.jpg](https://user-images.githubusercontent.com/67045883/154091601-78e0bee2-d274-4a13-88de-2ab6d3dc6c72.jpg)

Even after 2 hours on 1s lipo it's running fine, i deliberately kept backlight always on, still it's running fine. Please adjust lower voltage range from 3.5v

**Comment by ajjjjjjjj** [comment]
How about only battery warning going down to 3.5V?
This way it can be used by users like @TarasTadva who only want to see real voltage and have proper warning for it.
I don't like lowering overall range because it may work for you but what if internal RF want ramp up power which cannot be simulated in "at home" ranges? Hard to tell, and it's running beyond internal LDO specs.

**Comment by xpertcrush** [comment]
> How about only battery warning going down to 3.5V?
> This way it can be used by users like @TarasTadva who only want to see real voltage and have proper warning for it.
> I don't like lowering overall range because it may work for you but what if internal RF want to ramp up power which cannot be simulated in "at home" ranges? Hard to tell, and it's running beyond internal LDO specs anyway.
> 
> What do you think @marianomd ?

Totally agree, just lower the battery warning only. You are right about ldo specs and it is defying all the odds by running even at 3.4v which is strange as per my tests as well.

Also i test current consumption with display on and rf on it was between 100-130mah with occasional burst to 160 for a second when buzzer beeps.

**Comment by xpertcrush** [comment]
Tested today, voltage below have adverse effect on display when beeper consumes current. It flickers, so 4 volt is about right. Thank you ajjjjj for the efforts. You are right, radio can malfunction at lower voltage, it's not visible not but can suddenly show up. Current settings are good, no need to change anything, guys whoever wants to use one cell or 2 cell lipos do like what I did, i just soldered a phone battery 3000 mah with Li ion charging module with overload protection and step up boost converter to step voltage to 5.1v and it's working great.

Here is some pictures.

![P_20220216_195458.jpg](https://user-images.githubusercontent.com/67045883/154285824-e99c8833-9840-4333-b44c-cb6b6e62e00e.jpg)

![P_20220216_195503.jpg](https://user-images.githubusercontent.com/67045883/154285861-6dbd1682-8069-46be-b0ab-7a48c2739504.jpg)

![P_20220216_195509.jpg](https://user-images.githubusercontent.com/67045883/154285911-6c5e49ee-61de-4af0-98b6-a0b45120ddb2.jpg)

![P_20220216_200135.jpg](https://user-images.githubusercontent.com/67045883/154286044-7b0cd13e-9645-4139-bb9a-81362a48e2ca.jpg)

Two connectors which you see inside cabin are for external module and if i need to power something else with battery 🔋
